src/test/resources/features/LogIn.feature

```diff
@@ -0,0 +1,161 @@
+@LogIn
+  Feature: LogIn Scenarios
+
+@LogIn1
+  Scenario: LogIn Page with valid credentials
+  Given I navigate to "login" page
+  When I enter "lmislinski@nbobd.com" into email field
+  And I type password "123456"
+  When I hit button "Sign In"
+  And I wait for 4 seconds
+  When I click "Log Out"
+  Then confirmation window should appear
+  And I click "Log Out" button
+  Then return to "login"page
+
+    @LogIn2
+    Scenario: LogIn Page Register Now with valid credentials
```

Comment:
This test does not belong to this test set. Please remove

Reply:
Done